Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -5795,7 +5795,7 @@ script:
     - contextPath: Panorama.SecurityRule.Location
       description: The rule location.
       type: String
-    - contextPath: Panorama.SecurityRule.Category
+    - contextPath: Panorama.SecurityRule.CustomUrlCategory
```

Comment:
why?
you realize validation will fail on it? its backward compatibility break

Reply:
then you will need a force merge......

Reply:
i will anyway because of the "unsafe xml" thing